From a990bc796b7580d7ea2ab7954907acc5c1eea3e8 Mon Sep 17 00:00:00 2001
From: Sergey Vinokurov <serg.foo@gmail.com>
Date: Sat, 27 Dec 2025 00:47:57 +0000
Subject: [PATCH 1/2] Export header still used by ksysguard

---
 processcore/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/processcore/CMakeLists.txt b/processcore/CMakeLists.txt
index f97f17cc..fb266470 100644
--- a/processcore/CMakeLists.txt
+++ b/processcore/CMakeLists.txt
@@ -67,6 +67,7 @@ add_subdirectory(plugins)
 ########### install files ###############
 
 install( FILES
+    extended_process_list.h
     processes.h
     process.h
     process_controller.h
-- 
2.51.2

