From c2235cf1648b015f63bc38c63ef3f6afae526ea5 Mon Sep 17 00:00:00 2001
From: Sergey Vinokurov <serg.foo@gmail.com>
Date: Sat, 27 Dec 2025 01:01:21 +0000
Subject: [PATCH 2/2] Disable GPU and network plugins

---
 processcore/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/processcore/CMakeLists.txt b/processcore/CMakeLists.txt
index fb266470..04474d80 100644
--- a/processcore/CMakeLists.txt
+++ b/processcore/CMakeLists.txt
@@ -62,7 +62,7 @@ ecm_finalize_qml_module(processcore)
 
 install(TARGETS processcore EXPORT libksysguardLibraryTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS})
 
-add_subdirectory(plugins)
+# add_subdirectory(plugins)
 
 ########### install files ###############
 
-- 
2.51.2

